src/local-time.lisp

```diff
@@ -211,12 +211,12 @@
      finally
         (return (if (zerop start) 0 (1- start)))))
 
-(defun %subzone-as-of (timezone unix-time)
-  (let* ((as-of-time unix-time)
+(defun %subzone-as-of (timezone seconds days)
```

Comment:
Help me understand the utility of this. It doesn't seem more convenient for anything.

Reply:
There are two reasons:
1. Prevent allocations (as you guessed)
2. Use in the *code-universal-time-with-tz variants.